data/jyotir07.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "Jyotiraditya Singh",
+  "type": "personal",
+  "bio": "**Your Bio**",
+
+  "tags": ["WebDev", "tag2", "tag3"],
+
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/<Your Twitter Username>" },
+    { "icon": "FaGithub", "url": "https://github.com/<Your GitHub Username>" }, //for mini icons
+    {
+      "icon": "FaLinkedin",
+      "url": "Your Linkedin Profile URL"
+    }
+  ],
```

Add your Twitter, GitHub and LinkedIn usernames, if not, remove the section and also remove the comments (`//for mini icons`)


```suggestion
  "socials": [
    { "icon": "FaTwitter", "url": "https://twitter.com/<Your Twitter Username>" },
    { "icon": "FaGithub", "url": "https://github.com/<Your GitHub Username>" }, //for mini icons
    {
      "icon": "FaLinkedin",
      "url": "Your Linkedin Profile URL"
    }
  ],
```